safetab_p/README.md

```diff
@@ -93,18 +93,18 @@ export PYTHONPATH=/usr/lib/spark/python:/usr/lib/spark/python/lib:/usr/lib/spark
 `PYTHONPATH` needs to updated to include the Tumult source directories. This assumes that required Python dependencies have been installed.
 
 ```bash
-DIR=<absolute path of cloned tumult repository>
+DIR=<absolute path of cloned DAS_2020_DDHCA_Production_Code repository>
 export PYTHONPATH=$PYTHONPATH:$DIR/safetab_p
 export PYTHONPATH=$PYTHONPATH:$DIR/safetab_utils
-export PYTHONPATH=$PYTHONPATH:$DIR/core
-export PYTHONPATH=$PYTHONPATH:$DIR/common
-export PYTHONPATH=$PYTHONPATH:$DIR/analytics
+export PYTHONPATH=$PYTHONPATH:$DIR/tumult/core
+export PYTHONPATH=$PYTHONPATH:$DIR/tumult/common
+export PYTHONPATH=$PYTHONPATH:$DIR/tumult/analytics
 ```
 
 `PYTHONPATH` also needs to be updated to include a Census Edited File (CEF) reader module for SafeTab `safetab_cef_reader.py`. This can be either MITRE’s CEF reader (developed separately), or the built-in mock CEF reader: `safetab_p/tmlt/mock_cef_reader`. Note that the mock CEF reader does not actually read CEF files, so it can only be used if input is being read from CSV files.
 
 ```bash
-export PYTHONPATH=<absolute path to directory containing safetab_cef_reader.py>:$PYTHONPATH
+export PYTHONPATH=<absolute path of cloned DAS_2020_DDHCA_Production_Code repository>/mitre:$PYTHONPATH
 ```
 
 If using MITRE's CEF reader, you will also need to include its dependencies, which can be found in its `das_decennial` directory:
```

Comment:
We should probably update this bit as well, since we can be more specific now.

Reply:
After reviewing MITRE's requirements.txt file, we are removing this because it doesn't have any additional dependencies. It also appears to be incompatible with SafeTab-P because it tries to install a different numpy version. And their setup.py and poetry toml don't require pyspark, so things seem to be an issue there.

Our documentation has been updated here.